include/wx/window.h

```diff
@@ -411,6 +411,7 @@ class WXDLLIMPEXP_CORE wxWindowBase : public wxEvtHandler
         // minimum size, giving priority to the min size components, and
         // returns the results.
     virtual wxSize GetEffectiveMinSize() const;
+    virtual wxSize GetEffectiveMinSizeFirstPass() const { return  GetEffectiveMinSize(); }
```

Comment:
This should really be documented (in a comment here and in `interface/wx/window.h`) because I just have no idea how to use it right now.